tools/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+TAG=${TAG:-"v1.4.22"}
+
+cd tools >/dev/null 2>&1 || true
+
+docker build -t otomi/base:$TAG -f Dockerfile.base --squash .
+docker build -t otomi/tools:$TAG -f Dockerfile.tools --squash .
+# Dockerfile.cli is built from github workflow
+
+docker push otomi/base:$TAG
+docker push otomi/tools:$TAG
```

Comment:
Fixed  https://www.shellcheck.net/wiki/SC2148
```bash
#!/usr/bin/env bash
TAG=${TAG:-"v1.4.22"}

cd tools >/dev/null 2>&1 || true

docker build -t otomi/base:"$TAG" -f Dockerfile.base --squash .
docker build -t otomi/tools:"$TAG" -f Dockerfile.tools --squash .
# Dockerfile.cli is built from github workflow

docker push otomi/base:"$TAG"
docker push otomi/tools:"$TAG"
```

Reply:
Again, tags never contain spaces. So not needed. You might add exclusions for shellcheck which is a horrible tool